lib/main.js

```diff
@@ -47,6 +48,19 @@ const fixPathString = (pathString, fileDir, projectDir) => {
   return pRstring;
 };
 
+const resolvePath = (fullPath) => {
+  const paths = fullPath.split(';');
+  if (paths.length === 1) {
+    return paths[0];
+  }
+  for (let i = 0; i < paths.length; i += 1) {
+    if (fs().existsSync(paths[i])) {
+      return paths[i];
+    }
+  }
+  throw Error(`Path cannot be found: ${fullPath}`);
```

Comment:
A `throw`n error is considered a failure of the package, which prompts the user to file an issue. We **definitely** don't want users reporting an error here when they misconfigure their settings!

This should add an [error notification](https://atom.io/docs/api/v1.34.0/NotificationManager#instance-addError) and return a value that is handled in the caller location to know it should simply return `null` for the result of the entire `lint()` call (which tells the consumer not to update any messages it currently has saved.